7_config_merge/src/munge_lake_info.R

```diff
@@ -76,9 +76,9 @@ munge_Kw <- function(out_ind, kw_varying_ind, ...){
 #'
 #' For lakes that are parameterized with data for max depth,
 #' H and A will be two values, a zero area max depth, and a surface area zero depth
-munge_H_A <- function(out_ind, areas_ind, ...){
+munge_H_A <- function(out_ind, areas_ind, elev_ind, ...){
   lake_areas <- scipiper::sc_retrieve(areas_ind) %>% readRDS()
-  crest_height <- 320 # magic number for now, elevation of lakes, assumed average for now...
+  lake_elev <- scipiper::sc_retrieve(elev_ind) %>% arrow::read_feather()
```

Comment:
I had a long-winded Q here, but was able to resolve it myself by writing it down (and reviewing your overview)! This looks good 👍